pkg/decision/experiment_whitelist_service_test.go

```diff
@@ -122,6 +122,23 @@ func (s *ExperimentWhitelistServiceTestSuite) TestNoExperimentInDecisionContext(
 	s.Nil(decision.Variation)
 }
 
+// Test an experiment whose key differs from its id
+func (s *ExperimentWhitelistServiceTestSuite) TestWhitelistIncludesDecisionForExpWithKey() {
```

Comment:
Not sure we need a _new_ test for this or if we update the previous test fixture to have a differing id, key pair.

Reply:
Good point, I'll delete the new test and change the old test data.